tomls/omnibus-mainnet/bfp.toml

```diff
@@ -0,0 +1,290 @@
+
+[setting.incremental_margin_scalar]
+defaultValue = "<%= parseEther('1') %>"
+
+[setting.maintenance_margin_scalar]
+defaultValue = "<%= parseEther('0.3') %>"
+
+[setting.max_initial_margin_ratio]
+defaultValue = "<%= parseEther('0.9') %>"
+
+[setting.liquidation_reward_percent]
+defaultValue = "<%= parseEther('0.0005') %>"
+
+[setting.liquidation_limit_scalar]
+defaultValue = "<%= parseEther('1') %>"
+
+[setting.liquidation_max_pd]
+defaultValue = "<%= parseEther('0.0012') %>"
+
+[setting.liquidation_window_duration]
+defaultValue = "90"
+
+[setting.min_keeper_fee_usd]
+defaultValue = "<%= parseEther('10') %>"
+
+[setting.max_keeper_fee_usd]
+defaultValue = "<%= parseEther('100') %>"
+
+# --- Provision bfp market --- #
+
+[provision.bfp_market_factory]
+source = "<%= settings.bfp_market_package %>"
+targetPreset = "<%= settings.target_preset %>"
+options.coreProxyOwner = "<%= settings.owner %>"
+options.owner = "<%= settings.owner %>"
+options.salt = "<%= settings.salt %>"
+options.synthetixPackage = "<%= settings.snx_package %>@<%= settings.target_preset %>"
+options.synthetixPreset = "<%= settings.target_preset %>"
+depends = ["provision.system"]
+
+# Set pyth
+
+[invoke.BfpMarketProxy_setPyth]
+target = ["bfp_market_factory.BfpMarketProxy"]
+fromCall.func = "owner"
+func = "setPyth"
+args = ["<%= settings.pyth_price_verification_address %>"]
+
+# --- Add Rewards distributors --- #
+
+[invoke.BfpMarketProxy_createRewardDistributor_weth]
+target = ["bfp_market_factory.BfpMarketProxy"]
+fromCall.func = "owner"
+func = "createRewardDistributor"
+extra.weth_rewards_distributor.event = "RewardDistributorCreated"
+extra.weth_rewards_distributor.arg = 0
+
+[[invoke.BfpMarketProxy_createRewardDistributor_weth.args]]
+poolId = "<%= settings.sc_pool_id %>"
+name = "weth RewardDistributor"
+token = "<%= settings.weth_address %>"
+collateralTypes = [
+    "<%= settings.snx_address %>",
+    #    "<%= imports.wsteth.contracts.Token.address %>",
+    #    "<%= imports.susde.contracts.Token.address %>",
+]
+
+# --- Global Market configuration --- #
+
+[invoke.BfpMarketProxy_setMarketConfiguration]
+target = ["bfp_market_factory.BfpMarketProxy"]
+fromCall.func = "owner"
+func = "setMarketConfiguration"
+depends = ['provision.bfp_market_factory']
+
+[[invoke.BfpMarketProxy_setMarketConfiguration.args]]
+pythPublishTimeMin = 12
+pythPublishTimeMax = 60
+minOrderAge = 12
+maxOrderAge = 60
+minKeeperFeeUsd = "<%= settings.min_keeper_fee_usd %>"
+maxKeeperFeeUsd = "<%= settings.max_keeper_fee_usd %>"
+keeperProfitMarginUsd = "<%= settings.keeper_profit_margin_usd %>"
+keeperProfitMarginPercent = "<%= settings.keeper_profit_margin_percent %>"
+keeperSettlementGasUnits = 1200000
+keeperCancellationGasUnits = 600000
+keeperFlagGasUnits = 1200000
+keeperLiquidateMarginGasUnits = 1200000
+keeperLiquidationGasUnits = 1200000
+keeperLiquidationEndorsed = "<%= settings.deployer %>"                         # TODO, this is currently deployer, change to a keeper
```

Comment:
we need to resolve this todo

Reply:
Yes, this keeper address is whitelisted to exceed liquidation caps.